src/App.js

```diff
@@ -1,13 +1,63 @@
 import React from 'react';
+import { GoodsList } from './GoodsList';
 
 import './App.scss';
 
-// import { getAll, get5First, getRed } from './api/goods';
-// or
-// import * as goodsAPI from './api/goods';
+import { getAll, get5First, getRedGoods } from './api/goods';
 
-const App = () => (
-  <h1>Dynamic list of Goods</h1>
-);
+class App extends React.Component {
+  state = {
+    goods: [],
+  }
+
+  render() {
+    return (
+      <>
+        <h1>Dynamic list of Goods</h1>
+
+        <button
+          type="button"
+          name="loadAll"
+          onClick={() => {
+            getAll()
+              .then((goods) => {
+                this.setState({ goods });
+              });
+          }}
+        >
+          Load All goods
+        </button>
+
+        <button
+          type="button"
+          name="load5First"
+          onClick={() => {
+            get5First()
+              .then((goods) => {
+                this.setState({ goods });
+              });
+          }}
```

Comment:
```suggestion
          onClick={() => {this.show5Goods()}}
```

You can write this function outside render, and use it here